src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -236,6 +244,13 @@ def __init__(self, file_handle):
         """Initialize the object."""
         self._file_handle = file_handle
 
+    def get_mesh_type(self) -> MeshType:
```

Comment:
It's significant that the return value can be None.

Aside:: since we've dropped 3.9, it can now be written `MeshType | None`, which we _could_ apply everywhere over `Optional[]`.

Reply:
This particular case always returns "MeshType", right @seanpearsonuk ?

Reply:
Seems correct